package.json

```diff
@@ -55,12 +55,12 @@
     "react-draggable": "3.3.2",
     "react-emotion": "9.2.12",
     "react-focus-lock": "2.0.5",
+    "react-id-generator": "3.0.0",
```

Comment:
I decided to use a different library so we don't unnecessarily re-create the ID when a component re-renders